### PR TITLE
[fix][explorer ui] units for time elapsed

### PR DESCRIPTION
**Pull request by nautsimon**
https://x.com/defi_moses/status/1828110408223519157

## Summary by CodeRabbit

- **New Features**
  - Enhanced the transaction timing display with user-friendly elapsed time formatting.

- **Bug Fixes**
  - Improved accuracy in displaying time differences, ensuring clarity in feedback for users.

### CODE REVIEW

File: packages/explorer-ui/pages/tx/[kappa].tsx [hunk 1]
```diff
@@ -37,12 +37,14 @@ export const BridgeTransaction = ({ queryResult }) => {
   const transaction = queryResult.bridgeTransactions[0]
   const { pending, fromInfo, toInfo } = transaction
 
-  const getTimeDifference = (start, end) => {
-    const diff = end - start
-    if (0 >= diff) {
-      return '1'
+  // Get time taken to complete tx w/appropriate units.
```
Comment (CryptoMaxPlanck) [inline packages/explorer-ui/pages/tx/[kappa].tsx:40]:
```suggestion
  // Get time taken to complete tx w/ appropriate units.
```

Comment (nautsimon) [reply packages/explorer-ui/pages/tx/[kappa].tsx:40]: we can address this in a future PR, ill open a task for you

Comment (nautsimon) [reply packages/explorer-ui/pages/tx/[kappa].tsx]: fixed!

Comment (trajan0x) [reply packages/explorer-ui/pages/tx/[kappa].tsx]: LGTM

**Pull request merged**